yt/frontends/chombo/data_structures.py

```diff
@@ -732,13 +728,13 @@ def _is_valid(cls, filename, *args, **kwargs):
 
         if not pluto_ini_file_exists:
             try:
-                fileh = h5py.File(filename, mode="r")
-                valid = "CeilVA_mass" in fileh.attrs.keys()
-                valid = (
-                    "Chombo_global" in fileh["/"] and "Charm_global" not in fileh["/"]
-                )
-                valid = valid and "CeilVA_mass" in fileh.attrs.keys()
-                fileh.close()
+                with h5py.File(filename, mode="r") as fileh:
+                    valid = "CeilVA_mass" in fileh.attrs.keys()
+                    valid &= (
```

Comment:
here I'm actually trying to _fix_ a bug because I noticed that the first condition was actually irrelevant to the return value. Maybe the bug is that the line
```python
valid = "CeilVA_mass" in fileh.attrs.keys()
```
is _actually_ irrelevant and should be deleted
@atmyers might know better